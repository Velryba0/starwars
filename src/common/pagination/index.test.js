import React from 'react'
import { createShallow } from '@material-ui/core/test-utils'

import PaginationTable from './index'

describe('src/common/pagination', () => {
    let shallow;
  
    beforeEach(() => {
      shallow = createShallow(); 
    });
  
    it('Pass number', () => {
      const wrapper = shallow(<PaginationTable />);
      expect(wrapper).toMatchSnapshot();
    });
  });
