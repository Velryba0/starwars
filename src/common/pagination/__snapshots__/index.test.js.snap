// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/common/header/pagination Paginacion 1`] = `ShallowWrapper {}`;

exports[`src/common/header/pagination Pass number 1`] = `ShallowWrapper {}`;

exports[`src/common/header/pagination should work 1`] = `ShallowWrapper {}`;

exports[`src/common/pagination Pass number 1`] = `ShallowWrapper {}`;
